### PR TITLE
Overhaul

### LINKED ISSUE #91: Release 1.7.3 - Can no longer collapse Braindance Protocol interface when desired
## Description
Unable to collapse the Braindance Protocol interface as of release 1.7.3 (no triangle-like symbol to do so).

## Expected Behaviour
Attempts to circumvent this by editing 'Collapsed=0' to 'Collapsed=1' in layout.ini does not do so either.

### PR DESCRIPTION
**Pull request by Nats-ji**
Need test

Change list:
- File structure overhaul
- Added search bar
- Hotkeys now localizable
- Added favorites list
- Added hotkey list (bind any protocols with a hotkey)
- Brought back collapse #91 
- Auto update version number in manifest.json

image

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Looks absolutely amazing :) Great work Nats-ji 👍 
I've added a short / quick how-to configure the hotkeys, incase someone doesn't get it.

I've tested pretty much all I can ingame, and everything seems to work.

**Comment by ghost** [comment]
Updated the README with a little preview section and updated the how-to guide on custom hotkeys.
I say this is ready to merge 👍

File: mods/braindance_protocol/utility.lua [hunk 1]
```diff
@@ -1,11 +1,11 @@
 local Utilities = {}
 
 function Utilities.StartProtocol(name)
-    print(string.format("Executing Protocol: %s", name))
+    print(string.format("[BD] Executing Protocol: %s", name))
```
Comment (osulli) [inline mods/braindance_protocol/utility.lua]: I'm wondering if we can deprecate this as mentioned in #64 ?

**Review by osulli — APPROVED** [review]
Looks great

**Pull request merged**